third-party/realdds/src/dds-stream-base.cpp

```diff
@@ -36,6 +36,15 @@ void dds_stream_base::init_profiles( dds_stream_profiles const & profiles, int d
 }
 
 
+void dds_stream_base::init_options( dds_options const & options )
```

Comment:
Consider `&&` and `move`?

Reply:
Kept same interface as `init_profiles`. Some users keep profiles as values of a map (e.g. rs-dds-server) and I think that moving in middle of iterating might create problems.

Reply:
Overloaded the functions with `rvalue` versions to enable moving

Reply:
As discussed today, remove the overloads, keeping the `const &` version.